<?xml version="1.0"?>
<launch>

	<node name="lvr_slicer" pkg="lvr_ros" type="lvr_slicer_node" output="screen">
		<param name="verbose_lvr" value="false"/>
		<param name="activate_callback" value="true"/>
		<param name="mesh_topic" value="/triangle_mesh"/>
		<param name="single_slice_grid_topic" value="/single_grid"/>
		<param name="single_slice_marker_topic" value="/single_grid_marker"/>
		<param name="multi_slice_grid_topic" value="/multi_grid"/>
		<param name="multi_slice_marker_topic" value="/multi_grid_marker"/>

		<param name="dimension" value="z"/>
		<param name="single_slice_value" value="1.0"/>
		<param name="multi_slice_min_value" value="0.0"/>
		<param name="multi_slice_min_value" value="1.0"/>
		<param name="multi_slice_resolution" value="0.05"/>
		<param name="grid_resolution" value = "0.01"/>
		<param name="grid_unoccupied_default" value = "0.0"/>
	</node>

</launch>



